internal/resto/client.go

```diff
@@ -70,6 +74,40 @@ func (c *Client) PollJob(ctx context.Context, id string, interval time.Duration)
 	return job.Job{}, nil
 }
 
+// GetJobAssetFileContent returns the job asset file content.
+func (c *Client) GetJobAssetFileContent(ctx context.Context, id, fileName string) ([]byte, error) {
```

Comment:
what do you think about making the `id` parameter more descriptive, like `jobId`?

Reply:
Good idea!